{"ID": "398329b2-fbc0-4f7f-a23b-ea51f1ed9b70", "effective_cve": "CVE-2021-35939", "image": {"exact": "registry.access.redhat.com/ubi8@sha256:68fecea0d255ee253acbf0c860eaebb7017ef5ef007c25bee9eeffd29ce85b29"}, "label": "TP", "package": {"name": "python3-rpm", "version": "4.14.3-23.el8"}, "timestamp": "2022-11-01T22:35:23+00:00", "tool": "grype@v0.51.0", "user": "westonsteimel", "vulnerability_id": "CVE-2021-35939"}